src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/header.html

```diff
@@ -1,29 +1,36 @@
+{% if theme_navbar_start or theme_navbar_center or theme_navbar_end or theme_navbar_persistent %}
 <div class="bd-header__inner bd-page-width">
   <label class="sidebar-toggle primary-toggle" for="__primary">
     <span class="fa-solid fa-bars"></span>
   </label>
-  <div id="navbar-start">
+  {% if theme_navbar_start %}
+  <div id="navbar-start" class="navbar-header-items__start">
```

Comment:
why does this one get an `id` but many of the other start/end divs don't?

Reply:
So i found there was inconsistent behavior. In some cases we used id and in others we used a class that was more specific. I think in general it's good to use classes instead of id. Want me to change that?

Reply:
I'm happy to use classes unless there's something we must enforce to be unique (version switcher and search box are the only things coming to mind). So yeah, I'd keep the others as class-only and change this one to eliminate the `id`